core/globals.h

```diff
@@ -308,6 +308,17 @@ typedef struct _dr_stats_t {
     uint64 synchs_not_at_safe_spot;
 } dr_stats_t;
 
+/**
+ * Error codes of DR API routines.
+ */
+typedef enum {
+    DR_ERROR_NONE,
```

Comment:
Hmm so if this were a return value I would definitely want DR_ERROR_SUCCESS.  I think all the ext's have some kind of _SUCCESS, and so does WIndows:
https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-.  Did you deliberate pick sthg different b/c it's not a return value?  The Windows one isn't either.  I would still vote for _SUCCESS for consistency.

Reply:
Indeed. It is not a return value and I didn't want to assign 0 to a legitimate error code. Another option is to remove it completely and assign values to the enum, starting from 1. WDYT?

Reply:
I think you've misread my comment: I was asking to s/DR_ERROR_NONE/DR_ERROR_SUCCESS/.  I then rambled on about how maybe you wanted NONE instead of SUCCESS because it wasn't a direct return value, but that I would still vote for SUCCESS to match our other error enums and external things like Windows API.

Reply:
Yep, I understood you regarding the substitute.  I am now wondering whether or not you prefer that I remove this entry completed instead?

Reply:
But what would dr_get_error_code() return when there was no error?  Doesn't there need to be a no-error code?

Reply:
I was thinking of leaving it undefined. dr_get_error_code() should be used only upon a failed API routine therefore  DR_ERROR_NONE/DR_ERROR_SUCCESS _should_ never be returned. Consequently, I was wondering whether we should eliminate the entry from the user interface completely.

Reply:
Hmm, I see.  OK, starting from 1 seems reasonable.  Then we could add it back as 0 if we ever needed it for some reason.

Reply:
Done.